.github/workflows/website.yml

```diff
@@ -51,6 +51,7 @@ jobs:
           git status
           echo "::endgroup::"
 
+          git add -N .
```

Comment:
should we use same for `watch.yaml`?

Reply:
Sure, why not? 

But it is unlikely that the watch command generates a new file.

Reply:
Replace string by Enum class for instance..
But yeah, in that case, at least one file would probably changed too